wagtail/wagtailsearch/signal_handlers.py

```diff
@@ -22,16 +26,24 @@ def post_save_signal_handler(instance, **kwargs):
     indexed_instance = get_indexed_instance(instance)
 
     if indexed_instance:
-        for backend in get_search_backends(with_auto_update=True):
-            backend.add(indexed_instance)
+        for backend_name, backend in get_search_backends_with_name(with_auto_update=True):
+            try:
+                backend.add(indexed_instance)
+            except:
```

Comment:
Bare `except:` clauses are not a good idea, as I've previously discussed. Doing `except Exception:` is much safer. From PEP8:

>  A bare except: clause will catch SystemExit and KeyboardInterrupt exceptions, making it harder to interrupt a program with Control-C, and can disguise other problems. If you want to catch all exceptions that signal program errors, use `except Exception:` (bare except is equivalent to `except BaseException:`). 

Even better would be catching the specific errors raised. This could be done by requiring search backends to wrap backend-specific exceptions in a `SearchBackendException` class, much like Django does with its database layer, or by adding a `backend.add(indexed_instance, fail_silently=True)` option to the function.